index.html

```diff
@@ -0,0 +1,186 @@
+  <div class="yt2">
+    <h2>
+      Handcrafted, home-made masterpieces
+    </h2>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <input type="email" name="email" id="email" placeholder="Enter Your Email Id">
+      <br>
+      <input id="submit" type="submit" value="GET STARTED">
+
+    </form>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quidem ea laboriosam quia itaque deleniti praesentium, hic
+    at quam earum dolore necessitatibus dolor, non maxime cum distinctio libero rerum doloribus quisquam!
+    Accusamus animi facere excepturi nesciunt minima possimus fuga, quibusdam fugiat, doloribus, dolorum exercitationem
+    ab explicabo! Molestias, a corporis! Ratione assumenda culpa quaerat ullam, maxime inventore? Doloremque vero
+    quisquam amet ad.
+    Inventore, doloremque distinctio esse, voluptatibus ducimus itaque placeat sed vel odit cupiditate repellat, quidem
+    ipsum labore repudiandae sapiente nihil sint quaerat animi quia facilis quis! Consequuntur blanditiis nam quis
+    autem?
+    Quod, aliquid quisquam natus mollitia distinctio nobis vitae voluptates? Veritatis, dolores, omnis nam quidem qui
+    eius deserunt facere earum quam voluptates quibusdam aperiam itaque sed molestiae nobis sit commodi a.
+    Minus magnam illum placeat porro culpa voluptatem deserunt sint nisi eaque nulla possimus minima sequi, id est quo
+    nostrum. Aliquam minima quo quae. Cum, reprehenderit. Eum consequatur impedit vel ullam?
+    Tempore, modi placeat quia libero consequuntur asperiores optio nostrum cum voluptatem quasi aliquam. Quo iste odit
+    blanditiis obcaecati, placeat dolorem nostrum repellendus amet, fugit facilis cum! Maiores dignissimos earum nemo!
+    Doloremque ducimus dolor porro nisi. Sed praesentium exercitationem reprehenderit numquam sequi totam delectus vel,
+    rerum voluptatum quae enim assumenda hic. Saepe eligendi earum repudiandae id eum aliquam voluptatem facere
+    laudantium!
+    Dolor non placeat, quas voluptatum doloremque quisquam facere cum eum blanditiis ducimus saepe molestiae nesciunt
+    dolores. Vitae sequi id dolore distinctio illo, adipisci delectus minus? Culpa minus quae deserunt dolores.
+    Odit, accusantium animi eius itaque odio placeat officia aliquid recusandae cupiditate aspernatur modi reprehenderit
+    possimus non vitae ad quos impedit dolore accusamus ipsam facilis excepturi repellat! Laboriosam distinctio ratione
+    quis!
+    Doloremque quasi sequi placeat aperiam debitis maiores ducimus blanditiis quam inventore quod similique distinctio,
+    earum rerum architecto maxime eum assumenda laudantium doloribus nesciunt necessitatibus alias. Adipisci asperiores
+    repellendus provident consequuntur!
+    Modi molestias quo in omnis temporibus quod mollitia autem neque eveniet cum sed, sequi rerum quos hic asperiores?
+    Expedita enim quasi rerum praesentium eligendi incidunt autem explicabo voluptatibus quis optio!
```

Comment:
I think it would be better to put all this text inside a p tag as it is a paragraph

Reply:
- Kindly phrase your comments in a polite and considerate manner, like a suggestion, instead of a command. This will make it easy for the student to accept and implement them. :wink:

Reply:
Kindly put this text inside a p tag 😄 because this will make your HTML structure professional, as every thing in HTML is supposed to be inside a a tag 😉